libs/model/src/featurelayer.cpp

```diff
@@ -127,7 +254,10 @@ TileFeatureLayer::newFeatureId(
     const std::string_view& typeId,
     const KeyValuePairs& featureIdParts)
 {
-    // TODO: Validate ID parts
+    if (!validFeatureId(typeId, featureIdParts, true)) {
```

Comment:
```suggestion
    if (!validFeatureId(typeId, featureIdParts, false)) {
```

Reply:
I figured the feature id passed to this function should explicitly not include the prefix?

Reply:
Yes, hence in the other call above the flag is true 🔺